connman/plugins/sailfish_ofono.c

```diff
@@ -121,6 +125,8 @@ struct modem_data {
 	guint strength;
 	char *name;
 	char *imsi;
+	guint delayed_connect_id;
```

Comment:
I would consider naming it `delayed_set_connected`, `delayed_connect_notify` or something. I was confused by this naming.

Reply:
Well, it is the `id` of the event source and for the (modem_)`delayed_connect()` so.. for me "notify" would in ConnMan context indicate something to do with the notifier functionality.

Reply:
I guess `delayed_set_connected_id` is the compromise here.